check_photo_backups.py

```
@@ -0,0 +1 @@
+../check_photo_backups.py/check_photo_backups.py
```

Ruff (0.14.14)

[error] No such file or directory (os error 2)

(E902)